### PR TITLE
Unit tests

### LINKED ISSUE #6: Add unit tests for source code
Add unit tests to improve code quality

### PR DESCRIPTION
**Pull request by bigsergey**
Closes #6

### CODE REVIEW

File: .typo-ci.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+excluded_words:
+  - tsc
```
[the remark this answers is not in the document]
Comment (bigsergey) [reply .typo-ci.yml]: https://github.com/marketplace/typo-ci

File: src/__tests__/main.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+});
+
+test('should return error code when glob throws', async () => {
+  mockedGlob.mockReturnValue(Promise.reject(new Error()));
+
+  expect(await main()).toEqual(ERROR_CODE);
+});
+
+test('should return success code when all filenames are valid', async () => {
+  mockedGlob.mockReturnValue(Promise.resolve(['index.js', 'index.css']));
```
[the remark this answers is not in the document]
Comment (bigsergey) [reply src/__tests__/main.spec.ts]: i thought about it and at that moment I was lazy... I will mock it ;-)

File: src/__tests__/main.spec.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import main, { ERROR_CODE, SUCCESS_CODE } from '../main';
+
+const mockedGlob = (glob as unknown) as jest.Mock<Promise<string[]>>;
+
+test('should call glob with correct arguments', async () => {
+  mockedGlob.mockReturnValue(Promise.resolve([]));
+
+  await main();
+
+  expect(glob).toHaveBeenCalledTimes(ERROR_CODE);
```
[the remark this answers is not in the document]
Comment (bigsergey) [reply src/__tests__/main.spec.ts]: what I have done! 😱

[the remark this answers is not in the document]
Comment (bigsergey) [reply src/__tests__/main.spec.ts]: it was find and replace :-D

File: src/main.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import glob from 'fast-glob';
+
+import lintFiles from './lint-files';
+
+export const ERROR_CODE = 1;
```
[the remark this answers is not in the document]
Comment (bigsergey) [reply src/main.ts]: hm, I have found only 0 and 1 codes in documentation https://nodejs.org/api/process.html#process_process_exit_code.  🤔

[the remark this answers is not in the document]
Comment (erykpiast) [reply src/main.ts]: `1` means "some error occured". It's quite vague, you can do better than that :) http://tldp.org/LDP/abs/html/exitcodes.html#EXITCODESREF

[the remark this answers is not in the document]
Comment (bigsergey) [reply src/main.ts]: done 👍

**Pull request merged**